daemon/create.go

```diff
@@ -70,6 +76,39 @@ func (daemon *Daemon) ContainerCreate(job *engine.Job) error {
 	return nil
 }
 
+func validateVolumePath(config *runconfig.Config, hostConfig *runconfig.HostConfig) error {
+	//validate bind volums
+	for _, spec := range hostConfig.Binds {
```

Comment:
validation in parseBindMountSpec has been extracted to here, so maybe there's no need to reuse it

btw, sorry for mess up the branch as my peer is new to github, I'll do squash after everything is OK.